modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-running-as-root-user-namespace_{context}"]
+= Running buildah as root in a user namespace
+
+If you are using {pipelines-shortname} to run pods in the user namespace, you must pass the annotations that the CRI-O container engine expects. You can add these annotations for all users, or for a specific namespace.
+
```

Comment:
Are there any prerequisites for this procedure? If so, list them in a `.Prerequisites` section before the `.Procedure` section

Reply:
Modified the existing introduction, but I think a formal prerequisite for explicitly stating "logged in as a cluster administrator" would be an overkill. The requirement of being logged in as a cluster administrator is so pervasive and implied in almost everything that it is hardly ever stated explicitly.
I can't think of any other prerequisite.

Reply:
Procedures in the OCP monitoring docs always explicitly state what user role is required to do the procedure, whether a role with admin or just edit permissions, so in my experience it's not out of the ordinary to state it. However, if your area is only ever touched by users with admin rights, then I suppose it makes sense.